go/vt/schemadiff/table.go

```diff
@@ -799,3 +841,228 @@ func (c *CreateTableEntity) Drop() EntityDiff {
 	}
 	return &DropTableEntityDiff{from: c, dropTable: dropTable}
 }
+
+// apply attempts to apply an ALTER TABLE diff onto this entity's table definition.
+// supported modifications are only those created by schemadiff's Diff() function.
+func (c *CreateTableEntity) apply(diff *AlterTableEntityDiff) error {
+	if spec := diff.alterTable.PartitionSpec; spec != nil {
+		switch {
+		case spec.Action == sqlparser.RemoveAction && spec.IsAll:
+			// Remove partitioning
+			c.TableSpec.PartitionOption = nil
+		default:
+			return errors.Wrapf(ErrUnsupportedApplyOperation, "%v", sqlparser.String(spec))
```

Comment:
Can this be `errors.Wrap(ErrUnsupportedApplyOperation, sqlparser.String(spec))`?

Reply:
done